pkg/validation/validation_test.go

```diff
@@ -0,0 +1,42 @@
+package validation
```

Comment:
nit: please add license here.

Reply:
Added, but FYI we may resolve to remove all headers from the entire codebase with the changes to the governance model.

Also, we have many packages today without headers, so if we do remain with any kind of file header, we may want to think about a way to automate the addition/check of it.